userspace/falco/app_actions/load_rules_files.cpp

```diff
@@ -93,27 +93,41 @@ application::run_result application::load_rules_files()
 		falco_configuration::read_rules_file_directory(path, m_state->config->m_loaded_rules_filenames, m_state->config->m_loaded_rules_folders);
 	}
 
-	for (const auto& filename : m_state->config->m_loaded_rules_filenames)
+	std::vector<std::string> rules_contents;
+	falco::load_result::rules_contents_t rc;
+
+	try {
+		read_files(m_state->config->m_loaded_rules_filenames.begin(),
+			   m_state->config->m_loaded_rules_filenames.end(),
+			   rules_contents,
+			   rc);
+	}
+	catch(falco_exception& e)
+	{
+		return run_result::fatal(e.what());
+	}
+
+	for(auto &filename : m_state->config->m_loaded_rules_filenames)
 	{
 		falco_logger::log(LOG_INFO, "Loading rules from file " + filename + "\n");
 		std::unique_ptr<falco::load_result> res;
 
-		res = m_state->engine->load_rules_file(filename);
+		res = m_state->engine->load_rules(rc.at(filename), filename);
 
 		// Print the full output if verbose is true
 		if(m_options.verbose &&
 		   (!res->successful() || res->has_warnings()))
 		{
 			printf("%s\n",
```

Comment:
Big question: In the past, we used to print warnings and error on stderr when the verbose option was enabled (see: https://github.com/falcosecurity/falco/blob/f86423db76ce79917ae30499e17726e942a678b6/userspace/engine/rules.cpp#L467).

With the recent changes, we changed this to be stdout. Do we have a good reason for this? This is a change in the UX contract. I think it makes sense to print this info on stderr, specially since they are here just in verbose mode. 

The same applies for the `falco_engine::load_rules` method family.

Reply:
Yes, looking back at 0.31.1 (pre application actions change) and 0.32.2 (pre load rules result change) the behavior was:

load rules: 
  on error print to stderr (https://github.com/falcosecurity/falco/blob/0.31.1/userspace/falco/falco.cpp#L735)
  with verbose and warnings only print to stderr (https://github.com/falcosecurity/falco/blob/0.31.1/userspace/engine/rules.cpp#L467)

validate rules:
   on error print to both stdout and stderr (https://github.com/falcosecurity/falco/blob/0.31.1/userspace/falco/falco.cpp#L550)
    with verbose and warnings only print to stderr (https://github.com/falcosecurity/falco/blob/0.31.1/userspace/engine/rules.cpp#L467)
    
So this block should actually move below line 127, check for vebose, and only check for warnings.